runtime/helpers/APIhelpers.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+let log = global.log;
+
+module.exports = {
+/**
+	*  API call for GET, PUT, POST and DELETE functionality
+	* @param url
+	* @param method
+	* @param body
+	* @param fileName
+	* @param statusCode
+	* @type {{ GET: receive all info, POST: create, PUT: edit / update, DELETE: remove info }},
+	*/
+	apiCall: function (url, method, body, fileName, statusCode) {
```

Comment:
for what is this funktion? Integration tests should test the hole system (inlcuding frontend) and not only testing the api???

Reply:
This already was in Yulia's code. She might want to use it later on.

Reply:
this is just one function among many for concrete api test. That doesn't mean we use only such methods, but integrate them.

Reply:
so it is not used and api tests should not done in this repo, right? So just remove it?

Reply:
no please let it stay, we might need it in the future.